cms/static/js/views/group_configuration_details.js

```diff
@@ -84,31 +86,12 @@ function(BaseView, _, gettext, str) {
                         Translators: 'count' is number of units that the group
                         configuration is used in.
                     */
-                    'Used in %(count)s unit', 'Used in %(count)s units',
+                    'Used in {count} unit', 'Used in {count} units',
                     count
                 );
             }
 
-            return interpolate(message, { count: count }, true);
-        },
-
-        getOutlineAnchorMessage: function () {
-            var message = gettext(
-                    /*
-                        Translators: 'outlineAnchor' is an anchor pointing to
-                        the course outline page.
-                    */
-                    'This Group Configuration is not in use. Start by adding a content experiment to any Unit via the %(outlineAnchor)s.'
-                ),
-                anchor = str.sprintf(
-                    '<a href="%(url)s" title="%(text)s">%(text)s</a>',
-                    {
-                            url: this.model.collection.outlineUrl,
-                            text: gettext('Course Outline')
-                    }
-                );
-
-            return str.sprintf(message, {outlineAnchor: anchor});
+            return StringUtils.interpolate(message, { count: count });
```

Comment:
Same thought about removing the ``message`` variable to make this more concise and clearer as to what is being interpolated.